documentation/2.0/content/rcuinfo.md

```diff
@@ -1,10 +1,31 @@
 ### Specifying RCU connection information in the model
 
-During creating or updating a domain, there is new section `RCUDbInfo` in the model, under the `domainInfo` section, for specifying RCU database connection information without using the command-line arguments `-rcu_db` and `-rcu_prefix`. Use this to support a database where the connection string is more complex and requires extra options.
+When creating a JRF domain, you can provide all the RCU connection information in a section `RCUDbInfo` under the `domainInfo` section in the model.  
+It provides you with more flexibility over the basic command line arguments of 
+`-rcu_db` and `-rcu_prefix`.  Use this when the connection string is more complex and requires extra options.
+
+### Background on JRF domain RCU tables 
+
+A JRF domain creates several data sources from the JRF domain template.
+
+| Data source name        | JNDI name                  | Schema  | Target |
+|-------------------------|----------------------------|---------|---|
+| WLSSchemaDataSource     | jdbc/WLSSchemaDataSource   | prefix_WLS_RUNTIME | None |
+| LocalSvcTblDataSource   | jdbc/WLSSchemaDataSource   | prefix_STB | admin server |
+| opss-data-source        | jdbc/OpssDataSource        | prefix_OPSS | admin server and cluster |
+| opss-audit-viewDS       | jdbc/AuditViewDataSource   | prefix_IAU_VIEWER | admin server and cluster |
+| opss-audit-DBDS         | jdbc/AuditAppendDataSource | prefix_IAU_APPEND | admin server and cluster |
+| mds-owsm                | jdbc/mds/owsm              | prefix_MDS | admin server and cluster |
+
+By default, the JRF domain template data source only has the default information such as URL and schema with the default prefix `DEV`.  During domain creation,
+WDT will use the information you provided in the command line or in the `RCUDbinfo` section to override the default values from the template so that it can connect to the database you specified.
+
+For some advanced use case, such as using Oracle Active Grid Link data source or Multi data sources, you can provide a sparse model of the above data sources in a separate model file
```

Comment:
advanced use case -> advanced use cases
using Oracle -> using an Oracle
Active Grid Link -> Active GridLink data source
Multi data sources -> Multi Data Sources
of the above data sources -> of the data sources